massa-consensus/src/settings.rs

```diff
@@ -157,7 +159,14 @@ pub struct ConsensusSettings {
 }
 
 impl ConsensusSettings {
+    /// Utility method to derivate a full configuration (static + user defined) from an user defined one
+    /// This is really handy in tests where we want to mutate static defined values
     pub fn config(&self) -> ConsensusConfig {
+        // TODO: These assertion should be checked at compile time
+        // https://github.com/rust-lang/rfcs/issues/2790
```

Comment:
@adrien-zinger do you have a neat idea on how to do that? I could do a hacky thing of putting it in a `lazy_static`, it will still fail at runtime but "earlier".

Reply:
Ouuuuuuuh :grimacing: